# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/orage/hotfix-dbusdir.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- orage-4.21.0/meson_options.txt.vanilla	2026-09-09 17:25:37.318709681 +0300
+++ orage-4.21.0/meson_options.txt	2026-09-09 17:27:46.846313586 +0300
@@ -18,3 +18,10 @@
   value: false,
   description: 'Add X11 tray icon, not comptible with Wayland',
 )
+
+option(
+  'dbusdir',
+  type: 'string',
+  value: '/usr/share',
+  description: 'dbus-i dir',
+)
--- orage-4.21.0/meson.build.vanilla	2026-09-09 17:18:11.022874744 +0300
+++ orage-4.21.0/meson.build	2026-09-09 17:18:31.144689295 +0300
@@ -152,6 +152,7 @@
   '-DLOCALSTATEDIR="@0@"'.format(get_option('prefix') / get_option('localstatedir')),
   '-DSBINDIR="@0@"'.format(get_option('prefix') / get_option('sbindir')),
   '-DSYSCONFDIR="@0@"'.format(get_option('prefix') / get_option('sysconfdir')),
+  '-DDBUSDIR="@0@"'.format(get_option('dbusdir')),
   '-DHAVE_XFCE_REVISION_H=1',
 ]
 
@@ -192,7 +192,7 @@
   input: 'org.xfce.orage.service.in',
   output: 'org.xfce.orage.service',
   install: true,
-  install_dir: get_option('prefix') / get_option('datadir') / 'dbus-1' / 'services'
+  install_dir: get_option('dbusdir') / 'dbus-1' / 'services'
 )
 
 subdir('src')

